backend/src/main/java/hu/flowacademy/munkalap/exception/WorksheetUserException.java

```diff
@@ -0,0 +1,21 @@
+package hu.flowacademy.munkalap.exception;
+
+import org.springframework.http.HttpStatus;
+
+public class WorksheetUserException extends Exception {
```

Comment:
Származzon inkább a RuntimeException-ből, hogy ne kelljen megjelölni a függvénydefinícióban és try-catch-ben kezelni. A kezelésére implementálj egy osztályt `@ControllerAdvice`-szal.

Reply:
Ha ez egy előre elvárt hiba, akkor bizony kell neki exception, RuntimeException-t kezelni JUnit-ban en nem HP-s esetre nem szép. A service apihoz hozzátartozik, kényszeríti. hogy foglalkozzunk az errorkezeléssel is. Általában határozzuk meg, hogy hogyan legyen, szerintem nem review comment téme.

Reply:
Jogos, maradhat Exception az ős.